From: Richard Purdie <richard.purdie@linuxfoundation.org>
Subject: [PATCH 12/12] libtool.m4: For reproducibility stop encoding hostname in libtool script

For reproducibilty, stop encoding the hostname into the libtool script, this isn't
really adding much to debugging and most distros are carrying such a patch now as
reproducibility is important.

Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>

https://sources.debian.org/data/main/libt/libtool/2.4.6-10/debian/patches/
        no_hostname.patch

Upstream-Status: Submitted [https://lists.gnu.org/archive/html/libtool-patches/2021-10/msg00011.html]

---
Index: libtool-2.4.6/m4/libtool.m4
===================================================================
--- libtool-2.4.6.orig/m4/libtool.m4
+++ libtool-2.4.6/m4/libtool.m4
@@ -728,7 +728,6 @@ _LT_CONFIG_SAVE_COMMANDS([
     cat <<_LT_EOF >> "$cfgfile"
 #! $SHELL
 # Generated automatically by $as_me ($PACKAGE) $VERSION
-# Libtool was configured on host `(hostname || uname -n) 2>/dev/null | sed 1q`:
 # NOTE: Changes made to this file will be lost: look at ltmain.sh.
 
 # Provide generalized library-building support services.
